.github/workflows/integration_test.yml

```
@@ -0,0 +1,133 @@
+
+          echo "Waiting for chain to produce blocks..."
+
+          # wait for chain to start and check for 5 blocks
+          BLOCKS_FOUND=0
+          MAX_ATTEMPTS=60
+          ATTEMPT=0
+
+          while [ $BLOCKS_FOUND -lt 5 ] && [ $ATTEMPT -lt $MAX_ATTEMPTS ]; do
+            sleep 2
+            ATTEMPT=$((ATTEMPT+1))
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)